CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 Change Log
 ==========
 
+## Version 0.19.0
+
+_2019-7-29_
+
+### Swift
+
+ * Update to build with Swift 5.
+ * Allow sinks to be reused across render passes if a sink of the same action type is declared. (#443)
+ * Support updating the root workflow input on a WorkflowHost. (#351)
+ * Add support for RenderTester to expect child workflows and outputs. (#442)
```

Comment:
Nit: backtick `RenderTester` and other code names

Reply:
Updated - good call.